doc/3/controllers/realtime/subscribe/index.md

```diff
@@ -51,11 +51,3 @@ _Simple subscription to document notifications_
 _Subscription to document notifications with scope option_
 
 <<< ./snippets/document-notifications-leave-scope.java
-
-_Subscription to message notifications_
```

Comment:
why is this removed?

Reply:
Those lines are a mistake from the realtime controller PR, those snippets do not exists